R/touchstone_demographic_dataset.R

```diff
@@ -0,0 +1,245 @@
+# Get all touchstone demographic dataset rows for a set
+# of touchstones. Also lookup both strings and numerical
+# ids for demographic source and statistic type.
+
+db_tdd_for_touchstones <- function(con, touchstones) {
+  DBI::dbGetQuery(con, sprintf("
```

Comment:
would it be useful to put this (and other large SQL comments) into an external file or not?

Reply:
I think they'll be fairly specific to their context - these won't get any use outside of the touchstone_demographic_dataset world (I think). Will make a note to review this in a couple of PRs where I might have some more SQL to ponder. It does make the .R file a bit clumpy...